### PR TITLE
Add confluent cloud kafka integration notebook

### PR DESCRIPTION
**Pull request by oyeliseiev-ua**

### CODE REVIEW

File: notebooks/confluent-cloud-integration/notebook.ipynb [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Ingest data from Confluent Cloud (Kafka)"
+      ],
+      "metadata": {},
+      "id": "bfd1a1969826a2e7"
+    },
```
Comment (kesmit13) [inline notebooks/confluent-cloud-integration/notebook.ipynb:26]: Can you remove this title block? The title gets put into the header above automatically.

Comment (oyeliseiev-ua) [reply notebooks/confluent-cloud-integration/notebook.ipynb]: done

Comment (kesmit13) [reply notebooks/confluent-cloud-integration/notebook.ipynb]: I think you may have removed the wrong one. It looks like the generated header and footer are gone now.

Comment (kesmit13) [reply notebooks/confluent-cloud-integration/notebook.ipynb]: If you run the pre-commit checks again I think that should fix it.

Comment (oyeliseiev-ua) [reply notebooks/confluent-cloud-integration/notebook.ipynb]: Unable to run, I always got error: 

Traceback (most recent call last):
  File "/home/oyeliseiev-ua/Singlestore/Repositories/spaces-notebooks/resources/toml-check.py", line 5, in <module>
    import tomllib

I'm not experienced with python to fix it quickly

Comment (kesmit13) [reply notebooks/confluent-cloud-integration/notebook.ipynb]: You must have been able to do it once. The first version of your ipynb file was correct except for the addition of an extra header. If you go back to that version, I'll approve it and just fix that one issue after it's merged.

Comment (pmishchenko-ua) [reply notebooks/confluent-cloud-integration/notebook.ipynb:26]: this should be resolved, @kesmit13 could you please check?

**Review by kesmit13 — APPROVED** [review]

**Pull request merged**